extra.html

```diff
@@ -58,54 +58,45 @@
 
        <kubernetes-topology-icon kind="Pod">
          <svg class="kube-topology">
-           <use xlink:href="#vertex-Pod" x="15" y="15"></use>
+           <g transform="translate(15, 15)" class="Pod">
+             <circle r="15"></circle>
+             <text y="7">P</text>
```

Comment:
I tried to replace here the text with image element as done in manageiq (since png is used) and it seems not to work. it doesn't change the graph itself (where is now the graph nodes display defined?) and the legend's icon is not displayed (just a white circle)
does it happen to you as well?
example:
  <kubernetes-topology-icon kind="Pod">
         <svg class="kube-topology">
           <g transform="translate(15, 15)" class="Pod">
             <circle r="15"></circle>
             <image  x="-9" xlink:href="https://raw.githubusercontent.com/ManageIQ/manageiq/master/public/images/icons/new/container_group.png" y="-9"></image>
           </g>
         </svg>
       </kubernetes-topology-icon>
       <label>Pods</label>